src/creator.html

```diff
@@ -38,7 +39,29 @@ <h1 id="title" ng-bind="title" ng-click="dialog.edit = true"></h1>
 				Randomize Question Order: {{randomizeOrder === true ? "On" : "Off"}}
 			</button>
 		</div>
-
+		<div class="questionBankDialog-wrapper" ng-show= "questionBankDialog">
+			<div class="questionBankDialog">
+				<div class="enable-qb-question">
+					<strong>Enable question bank? </strong>
```

Comment:
Since this is associated with an input selection, prefer `<label>`. Remember that semantic HTML is always ideal.